### PR TITLE
 #303: Fixed AWS Codebuild

### LINKED ISSUE #303: AWS Code Build fails with error message
[AWS CodeBuild Log](https://eu-central-1.console.aws.amazon.com/codesuite/codebuild/561494727831/projects/DataScienceSandboxCICodeBui-S5qWsq3zRyLR/build/DataScienceSandboxCICodeBui-S5qWsq3zRyLR%3A3ef2e7ee-bf60-4a24-ba6a-f3c52712cb83/?region=eu-central-1)

In setup of `test_jupyter_with_ec2_based_on_new_ami` 
```python
    @pytest.fixture(scope="session")
    def new_ec2_from_ami():
```

[Line 97](https://github.com/exasol/ai-lab/blob/main/test/codebuild/test_ci.py#L97)
  
```python
change_password(host=ec2_instance_description.public_dns_name, user='ubuntu',
                                curr_pass=default_password, new_password=new_password)
``` 

Error message

> /root/.cache/pypoetry/virtualenvs/exasol-ai-lab-X0J5PNhP-py3.10/lib/python3.10/site-packages/
> paramiko/auth_handler.py:263: BadAuthenticationType

### PR DESCRIPTION
**Pull request by ckunki**
Closes #303

### CODE REVIEW

**Review by kaklakariada — DISMISSED** [review]

File: exasol/ds/sandbox/runtime/ansible/reset_password_tasks.yml [hunk 1]
```diff
@@ -1,16 +1,16 @@
-- name: set new default password
+- name: Set New Default Password
   ansible.builtin.user:
     name: "{{ansible_user}}"
     state: present
-    password: "{{ default_vm_password | password_hash('sha512') }}"
+    password: "{{ default_vm_password }}"
```
Comment (tkilias) [inline exasol/ds/sandbox/runtime/ansible/reset_password_tasks.yml:5]: Maybe, we should rename this variable to default_vm_password_hash

File: exasol/ds/sandbox/runtime/ansible/reset_password_tasks.yml [hunk 2]
```diff
@@ -19,20 +19,25 @@
     state: present
     backrefs: yes
 
-- name: Enable SSH password authentication
+- name: Find SSH Config Files
+  shell: ls /etc/ssh/sshd_config.d/* /etc/ssh/sshd_config
+  register: ssh_config_files
+
+- name: Enable SSH Password Authentication
```
Comment (tkilias) [inline exasol/ds/sandbox/runtime/ansible/reset_password_tasks.yml:26]: Where isnthe kbinteractive

File: exasol/ds/sandbox/runtime/ansible/reset_password_tasks.yml (hunk 1 above)
Comment (ckunki) [reply exasol/ds/sandbox/runtime/ansible/reset_password_tasks.yml:5]: Yes, see next push.

File: exasol/ds/sandbox/runtime/ansible/reset_password_tasks.yml [hunk 3]
```diff
@@ -19,20 +19,25 @@
     state: present
     backrefs: yes
 
-- name: Enable SSH password authentication
+- name: Find SSH Config Files
+  shell: ls /etc/ssh/sshd_config.d/* /etc/ssh/sshd_config
+  register: ssh_config_files
```
Comment (tomuben) [inline exasol/ds/sandbox/runtime/ansible/reset_password_tasks.yml:24]: Why not using the builtin Ansible find task:
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/find_module.html

File: test/notebooks/nbtest_sagemaker.py [hunk 4]
```diff
@@ -182,6 +182,7 @@ def continuous_job_polling():
     )
 
 
+@pytest.mark.skip(reason="Want to get the results for other tests quickly")
```
Comment (kaklakariada) [inline test/notebooks/nbtest_sagemaker.py:185]: Did you mean to remove this?

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**